include/deal.II/base/partitioner.h

```diff
@@ -93,10 +157,12 @@ namespace Utilities
      * present class. A VectorOperation::values flag can be passed to decide
      * on how to combine the data in the ghost field with the data at the
      * owner, since both relate to the same data entry. In assembly, this is
-     * usually and add-into operation. This call starts non-blocking MPI
-     * communication routines, but does not wait for the routines to
-     * finish. Thus, the user may not write into the respective positions of
-     * the underlying arrays as the data might still be needed by MPI.
+     * usually and add-to operation where contributions from all processes to
```

Comment:
```suggestion
     * usually an add-to operation where contributions from all processes to
```
(pre-existing, but let us fix this anyway)